R/utils.R

```diff
@@ -271,12 +270,14 @@ include_css_files <- function(pattern = "*") {
   return(shiny::singleton(shiny::tags$head(lapply(css_files, shiny::includeCSS))))
 }
 
-
-#' Get a string with java-script code checking if the specific tab is clicked
-#' @description will be the input for `shiny::conditionalPanel()`
+#' JavaScript condition to check if a specific tab is active
+#'
 #' @param id `character(1)` the id of the tab panel with tabs.
 #' @param name `character(1)` the name of the tab.
+#' @return JavaScript string for use in `shiny::conditionalPanel()` to determine
```

Comment:
"JavaScript string" seems weird for what the function returns. Technically, it just returns a string which is a valid JS code.

Something like this would be nice:
```r
#' @return `string` with a valid JS code to be used in `shiny::conditionalPanel()` to determine
```

Reply:
![image](https://github.com/insightsengineering/teal.modules.general/assets/114988527/0cf28a78-c66b-4c5a-b883-5728e3467123)

Reply:
Fair enough. Please change it to `JavaScript expression`
```suggestion
#' @return JavaScript expression to be used in `shiny::conditionalPanel()` to determine
```